content/07.pathogenesis.md

```diff
@@ -26,3 +26,22 @@ This section would also be great for the introduction of zoonotic diseases which
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
 [Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies. 
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases (Hood and Tian, 2012, PMID: 23084773). 
```

Comment:
The next step will be adding in the Manubot-style automatic references.  For example, this manuscript has a DOI, which you can get from PubMed or the journal website
![image](https://user-images.githubusercontent.com/3532898/77592593-7c83a580-6ec0-11ea-8870-d2fe1e773a90.png)

The DOI is the best identifier to use if the article has one.  You can replace the citation with the Manubot DOI citation syntax:
```suggestion
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004]. 
```

Now Manubot will automatically pull in the reference details!

Reply:
Many thanks for this suggestion. Please could you suggest how do it when there are more than one references? Are those need to be separated by , or ;

Reply:
I have resolved the issue. Please have a look.